### PR TITLE
fix wrong address

### PR DESCRIPTION
**Pull request by danoctavian**
Upon a previous upgrade erroneous changes were made to the deployment files and new implementations updated the wrong file fields.

Also json field names are updated to work with the verifier scripts as shown below:

To check deployment integrity run:

```make ynEigen-verify  path=script/ynEigen/input/lsd-holesky.json rpc="https://rpc.ankr.com/eth_holesky" deployer=0x72fdbd51085bda5eeed3b55d1a46e2e92f0837a5 api=[etherscan-apikey]```

``` make ynEigen-verify  path=script/ynEigen/input/lsd-mainnet.json rpc="[mainnet-rpc]"  deployer=0x72fdbd51085bda5eeed3b55d1a46e2e92f0837a5 api=[etherscan-apikey]```

### CODE REVIEW

File: deployments/YnLSDe-1.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
   "proxy-eigenStrategyManager": "0x92D904019A92B0Cafce3492Abb95577C285A68fC",
   "proxy-rateProvider": "0xb658Cf6F4C232Be5c6035f2b42b96393089F20D9",
   "proxy-tokenStakingNodesManager": "0x6B566CB6cDdf7d140C59F84594756a151030a0C3",
-  "proxy-ynEigenDepositAdapter": "0xA9CD5a5199177b704287e379e85af95F0f061Bf2",
+  "proxy-ynEigenDepositAdapter": "0x9e72155d301a6555dc565315be72D295c76753c0",
```
Comment (danoctavian) [inline deployments/YnLSDe-1.json:26]: this was set to the implementation by accident

File: deployments/YnLSDe-17000.json [hunk 2]
```diff
@@ -25,10 +25,10 @@
   "proxy-tokenStakingNodesManager": "0x5c20D1a85C7d9acB503135a498E26Eb55d806552",
   "proxy-ynEigenDepositAdapter": "0x7d0c1F604571a1c015684e6c15f2DdEc432C5e74",
   "proxy-ynEigenViewer": "0xECCE549926637E974Ca5739a8827E612682415E7",
-  "proxyAdmin-YnLSDe": "0x92Df7b6001EF4cFe40B6b425bC4a358c05f27624",
+  "proxyAdmin-YnLSDe": "0x31456Eef519b7ab236e3638297Ed392390bf304F",
```
Comment (danoctavian) [inline deployments/YnLSDe-17000.json:28]: this was set to the implementation of ynLSDe

File: deployments/YnLSDe-17000.json [hunk 3]
```diff
@@ -25,10 +25,10 @@
...
   "proxyAdmin-assetRegistry": "0x4248392db8Ee31aA579822207d059A28A38c4510",
   "proxyAdmin-eigenStrategyManager": "0x010c60d663fddDAA076F0cE63f6692f0b5605fE5",
-  "proxyAdmin-rateProvider": "0x401af3714dc0c0d28f752dbdd70cf423c667246e",
+  "proxyAdmin-rateProvider": "0xD7E36d7f08ea756bb699B679B49F3095Cb6F7AB8",
```
Comment (danoctavian) [inline deployments/YnLSDe-17000.json:31]: this was set to a previous implementation of rateProvider

File: deployments/YnLSDe-17000.json [hunk 4]
```diff
@@ -11,24 +11,24 @@
   "UNPAUSE_ADMIN": "0x743b91CDB1C694D4F51bCDA3a4A59DcC0d02b913",
   "YNDev": "0x9Dd8F69b62ddFd990241530F47dcEd0Dad7f7d39",
   "YnSecurityCouncil": "0x743b91CDB1C694D4F51bCDA3a4A59DcC0d02b913",
-  "implementation-YnLSDe": "0xA6cf070663127FAa41a2150BdadAD8D84b876AEb",
+  "implementation-ynLSDe": "0x92Df7b6001EF4cFe40B6b425bC4a358c05f27624",
   "implementation-assetRegistry": "0x7385f1dBF3b0Ee3d85c89A38901Aa87333D30946",
   "implementation-eigenStrategyManager": "0xc599064e66FB509FD49B5083896649a742066758",
-  "implementation-rateProvider": "0x94009d1296dc2e6370Aa34c94BEDFeD55f96a276",
+  "implementation-rateProvider": "0x0Ca50907B8B5fDc6b0B1d5F23195209aD0457113",
```
Comment (danoctavian) [inline deployments/YnLSDe-17000.json:17]: latest implementation

File: deployments/YnLSDe-17000.json [hunk 5]
```diff
@@ -11,24 +11,24 @@
   "UNPAUSE_ADMIN": "0x743b91CDB1C694D4F51bCDA3a4A59DcC0d02b913",
   "YNDev": "0x9Dd8F69b62ddFd990241530F47dcEd0Dad7f7d39",
   "YnSecurityCouncil": "0x743b91CDB1C694D4F51bCDA3a4A59DcC0d02b913",
-  "implementation-YnLSDe": "0xA6cf070663127FAa41a2150BdadAD8D84b876AEb",
+  "implementation-ynLSDe": "0x92Df7b6001EF4cFe40B6b425bC4a358c05f27624",
```
Comment (danoctavian) [inline deployments/YnLSDe-17000.json:14]: latest implementation

File: deployments/YnLSDe-17000.json [hunk 6]
```diff
@@ -11,24 +11,24 @@
...
-  "proxy-YnLSDe": "0x071bdC8eDcdD66730f45a3D3A6F794FAA37C75ED",
+  "proxy-ynLSDe": "0x071bdC8eDcdD66730f45a3D3A6F794FAA37C75ED",
   "proxy-assetRegistry": "0xaD31546AdbfE1EcD7137310508f112039a35b6F7",
   "proxy-eigenStrategyManager": "0xA0a11A9b84bf87c0323bc183715a22eC7881B7FC",
   "proxy-rateProvider": "0xd68C29263F6dC2Ff8D9307b3AfAcD6D6fDeFbB3A",
   "proxy-tokenStakingNodesManager": "0x5c20D1a85C7d9acB503135a498E26Eb55d806552",
   "proxy-ynEigenDepositAdapter": "0x7d0c1F604571a1c015684e6c15f2DdEc432C5e74",
   "proxy-ynEigenViewer": "0xECCE549926637E974Ca5739a8827E612682415E7",
-  "proxyAdmin-YnLSDe": "0x92Df7b6001EF4cFe40B6b425bC4a358c05f27624",
+  "proxyAdmin-ynLSDe": "0x31456Eef519b7ab236e3638297Ed392390bf304F",
```
Comment (danoctavian) [inline deployments/YnLSDe-17000.json:28]: this was set to the implementation, not the proxy erroneously

File: script/ynEigen/input/lsd-mainnet.json [hunk 7]
```diff
@@ -19,9 +19,9 @@
       "strategy": "0xa4C637e0F704745D182e4D38cAb7E7485321d059"
     },
     {
-      "name": "RETH",
-      "token": "0xae78736Cd615f374D3085123A210448E74Fc6393",
-      "strategy": "0x1BeE69b7dFFfA4E2d53C2a2Df135C388AD25dCD2"
+      "name": "METH",
```
Comment (danoctavian) [inline script/ynEigen/input/lsd-mainnet.json:22]: after update by RaidGuild this descriptor file checked for RETH instead of METH - updated.

**Review by gearcat0 — APPROVED** [review]

**Review by xhad — APPROVED** [review]

**Pull request merged**